sktime/regression/kernel_based/_rocket_regressor.py

```diff
@@ -203,8 +203,7 @@ def _get_delegate(self):
 
     @classmethod
     def get_test_params(cls, parameter_set="default"):
-        """Return testing parameter settings for the estimator.
-
+        """Return testing parameter settings for the estimator
```

Comment:
Why are we removing the period at line end and the new line? Did you check pre-commit locally? Are you not getting errors from pydocstyle?